network.bisq.Bisq.yml

```diff
@@ -0,0 +1,86 @@
+    - name: bisq
+      buildsystem: simple
+      build-commands:
+          - gpg --import publickey.gpg; gpg --verify Bisq.deb.asc Bisq.deb
+          - ar x Bisq.deb
+          - rm Bisq.deb
+          - tar xf data.tar.xz
+          - cp -r opt/bisq/lib /app/lib
+          - install -D opt/bisq/bin/Bisq /app/bin/Bisq
+      sources:
+          - type: file
+            sha256: 310fa6381914d327ae1c1da995a78cad01df7f9a9b15863ad257f55194af9457
+            url: "https://bisq.network/downloads/v1.7.5/Bisq-64bit-1.7.5.deb"
+            dest-filename: Bisq.deb
+            only-arches:
+                - x86_64
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: "https://bisq.network/downloads/v$version/Bisq-64bit-$version.deb"
+          - type: file
+            url: https://bisq.network/downloads/v1.7.5/Bisq-64bit-1.7.5.deb.asc
+            dest-filename: Bisq.deb.asc
+            sha256: 84a634b4d5ad3d89c1130759f75cdb3e93c324b35538f8cdbd762ef6865ee929
+            only-arches:
+                - x86_64
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: https://bisq.network/downloads/v$version/Bisq-64bit-$version.deb.asc
+          - type: file
+            url: https://bisq.network/downloads/v1.7.5/29CDFD3B.asc
+            dest-filename: publickey.gpg
+            sha256: 61a87454943ab2229eb3f8b00e5a8558aa1ff7735af1f8f9bcf390d059832df3
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: https://bisq.network/downloads/v$version/29CDFD3B.asc
+    - name: icon
+      buildsystem: simple
+      build-commands:
+          - mkdir -p /app/share/icons/hicolor/512x512/apps
+          - cp icon_256x256@2x.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      sources:
+          - type: file
+            url: https://github.com/bisq-network/bisq/raw/v1.7.5/desktop/package/macosx/Bisq.iconset/icon_256x256%402x.png
+            sha256: 55b66ca4fe184a2916f7d72e910bc24b48e3e959cd88ad08767fc76371423f84
+            dest-filename: icon_256x256@2x.png
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: https://github.com/bisq-network/bisq/raw/v$version/desktop/package/macosx/Bisq.iconset/icon_256x256%402x.png
+    - name: desktop-entry
+      buildsystem: simple
+      build-commands:
+          - install -D network.bisq.Bisq.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      sources:
+          - type: file
+            path: network.bisq.Bisq.desktop
+    - name: metadata
+      buildsystem: simple
+      build-commands:
+          - install -D network.bisq.Bisq.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      sources:
+          - type: file
+            path: network.bisq.Bisq.appdata.xml
```

Comment:
It's not like it particularly matters, but these can definitely be a single module.

Reply:
Yeah but since they usually don't change together (and they don't raise errors together), it's more convenient to separate them